src/groups/mqb/mqbnet/mqbnet_channel.cpp

```diff
@@ -86,7 +85,7 @@ Channel::Channel(bdlbb::BlobBufferFactory* blobBufferFactory,
 , d_name(name, d_allocator_p)
 , d_stats()
 {
-    bslmt::ThreadAttributes attr = bmqsys::ThreadUtil::defaultAttributes();
+    bslmt::ThreadAttributes attr = bslmt::ThreadAttributes();
```

Comment:
```suggestion
    bslmt::ThreadAttributes attr;
```
Possible to simplify